app/src/main/java/com/imashnake/animite/features/MainActivity.kt

```diff
@@ -122,7 +124,6 @@ fun MainScreen(modifier: Modifier = Modifier) {
                     bottom = searchBarBottomPadding
                 )
                 .navigationBarsPadding(),
-            navController = navController
```

Comment:
This sparks joy, but where is click handling now?

Reply:
*Not yet*